src/components/meetups/MeetupRatingPage.tsx

```diff
@@ -87,45 +87,47 @@ export default function MeetupRatingPage({
   const [showDialog, setShowDialog] = useState(false);
   return (
     <Box backgroundColor="mainBackground" style={styles.root}>
-      <Box style={{ width: "100%" }}>
-        <MeetupCard
-          id={meetupID}
-          timestamp={timestamp}
-          duration={duration}
-          location={location}
-          userList={userList}
-        />
-        <ThemedCard>
-          <Box style={{ width: "100%", alignItems: "center" }}>
-            <Text variant="body">How would you rate your experience?</Text>
-            <AirbnbRating /* @ts-ignore:disable-next-line */
-              reviewColor={theme.colors.iconPrimary}
-              defaultRating={rating}
-              selectedColor={theme.colors.iconPrimary}
-              onFinishRating={setRating}
-            />
-            <Box mt="md">
-              <ThemedButton title="Submit Rating" onPress={submitRating} />
+      <ScrollView>
+        <Box style={{ width: "100%" }}>
+          <MeetupCard
+            id={meetupID}
+            timestamp={timestamp}
+            duration={duration}
+            location={location}
+            userList={userList}
+          />
+          <ThemedCard>
+            <Box style={{ width: "100%", alignItems: "center" }}>
+              <Text variant="body">How would you rate your experience?</Text>
+              <AirbnbRating /* @ts-ignore:disable-next-line */
```

Comment:
what error was this throwing?

Reply:
This is included because the type/props for AirbnbRating are not up to date. There was a prop that was in the docs and worked when I added the code, that was throwing a invalid prop type.